### PR TITLE
Add presubmits for slack-infra

### PR DESCRIPTION
**Pull request by Katharine**
whee

### CODE REVIEW

**Comment by Katharine** [comment]
This helpfully got autoassigned to people who aren't around.

/cc @cjwagner @fejta

File: config/jobs/kubernetes-sigs/slack-infra/OWNERS [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+- Katharine
```
Comment (fejta) [inline config/jobs/kubernetes-sigs/slack-infra/OWNERS]: Add at least one other person please

File: config/jobs/kubernetes-sigs/slack-infra/slack-infra-config.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
+presubmits:
+  kubernetes-sigs/slack-infra:
+  - name: pull-slack-infra-verify
+    decorate: true
+    path_alias: sigs.k8s.io/slack-infra
+    always_run: true
+    spec:
+      containers:
+        - image: gcr.io/k8s-testimages/kubekins-e2e:v20190329-811f7954b-master
```
Comment (fejta) [inline config/jobs/kubernetes-sigs/slack-infra/slack-infra-config.yaml]: Can we use a less horrible image? What exactly from this image do you need? Seems like you could just as well just use a golang image

**Pull request merged**